enterprise/admin/actions.py

```diff
@@ -59,3 +63,34 @@ def export_as_csv(modeladmin, request, queryset):  # pylint: disable=unused-argu
 
     export_as_csv.short_description = description
     return export_as_csv
+
+
+def refresh_catalog(self, request, queryset):  # pylint: disable=unused-argument
+    """
+    Kicks off background running tasks for refreshing catalogs
+    """
+    catalog_client = EnterpriseCatalogApiClient(user=request.user)
+    refreshed_catalogs, failed_to_refresh_catalogs = catalog_client.refresh_catalogs(queryset)
+
+    # display catalog and task ids that were successfully started refreshing
+    updated_message = ''
```

Comment:
It probably simplifies a lot if we restrict the updates to a single catalog, but the string concatenation for updated_message is hard to follow and read for me. It's not perfectly DRY but might be cleaner to write out the admin message in larger chunks?

Reply:
I also think it could be helpful to throw the task id into the success message if we can. Doesn't matter for ECS but saves a little digging for any developer who winds up doing this and then wants to check flower/splunk to follow the progress

Reply:
Task id is in the success message. I posted screenshots of what the messages look like in the PR descriptions.

Reply:
Ah nice, I missed it at the bottom. Ignore that comment then